packages/monitors/README.md

```diff
@@ -10,6 +10,14 @@ For more information about running a monitor bot, see the [docs](https://docs.um
 yarn add @uma/monitors
 ```
 
+## Running the monitors
+
+The simplest way to run the monitors is:
+
+```bash
+EMP_ADDRESS=0x1234 CUSTOM_NODE_URL=https://your.node.url.io MNEMONIC="your mnemonic (12-word seed phrase) here" MONITOR_CONFIG="{your monitor config object}" monitors --network mainnet_mnemonic
```

Comment:
What's the MVP MonitorConfig to get this to work?

Reply:
Not sure yet! I should be able to put together an MVP config during the EMP war games, which will give me a better handle on the monitor bots.

Reply:
Looks like a `MONITOR_CONFIG` is optional.